demo/portal/src/app/main.js

```diff
@@ -28,11 +30,16 @@ angular.module('app', [
     mainColumnViewId: "content@app",
     stepDefinitions: [
       vehicleInspectionStep.definition,
-      accidentStep.definition
+      accidentStep.definition,
+      signatureStep.definition,
+      feedbackStep.definition
     ]
   }),
+  // Load steps for preview
   vehicleInspectionStep.ngModule(),
-  accidentStep.ngModule()
+  accidentStep.ngModule(),
+  signatureStep.ngModule(),
+  feedbackStep.ngModule
```

Comment:
I made this a function in the other modules so creating the actual ng module doesn't get run if it doesn't need to.

We can still change the structure since the goal is just to having the `definition` on the main export.

Reply:
Nevermind, I see this is part of the portal app source, deriving from the dynamic forms step, so the user would have control of that, awesome!